src/NearOrg/Notifications/NotificationsList.jsx

```diff
@@ -16,6 +16,8 @@ const index = {
   },
 };
 
+let { manageNotification } = props;
```

Comment:
Let's move this to line [4](https://github.com/near/near-discovery-components/pull/520/files#diff-3665dbffa6eb0e091687824986cb9b62bdc720ceb6d9dccf8bdb38d59d67d9b2R4) so all the props will be at the same place at the top

Reply:
This has been addressed in the latest commit. Thank you